_data/governments.yml

```diff
@@ -639,20 +634,17 @@ U.S. Federal:
   - ntia
   - ombegov
   - Open-Sat
-  - opengovplatform
-  - ozoneplatform
```

Comment:
you sure on ozone being a private company?

Do you have new intel on that?

Reply:
See comment below on ozoneplatform

Reply:
Eh, it's a bit more complicated than that. it started as a Govt off-the shelf project, was put into the public sphere as part of the [FY2012 NDAA (sec. 924)](https://www.gpo.gov/fdsys/pkg/PLAW-112publ81/pdf/PLAW-112publ81.pdf), was then managed by a Government Open Source Managed board (GOSS as it was called at the time), but has recently been transitioned to Next Century as custodian of the efforts. In reality, I think the framework is mostly deprecated and no longer used by many in the DoD/IC community, but I think through the 
 Cooperative Research and Development Agreements (CRADA) that denotes Next Century (ozoneplatform.org) as the custodian of Ozone, I think it's still :+1:.

That said, i'll let you make the final decision.

Reply:
Since it goes to a .com and they are selling commercial services, I think we should take them out of the US gov't list.

Reply:
fair enough